utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,63 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.LevelListDrawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import java.net.URL
+
+/** UrlImage Parser for android TextView to load Html Image tag.  */
+class UrlImageParser(internal var tvContents: TextView, internal var context: Context) : Html.ImageGetter {
+
+  var targets: ArrayList<BitmapTarget>? = null
+  /***
+   * Get the Drawable from URL
+   * @param urlString
+   * @return
+   */
+  override fun getDrawable(urlString: String): Drawable {
+    val urlDrawable = UrlDrawable()
+    val load = Glide.with(context).asBitmap().load(URL(urlString))
+    val target = BitmapTarget(urlDrawable)
+    targets?.add(target)
+    load.into(target)
+    return urlDrawable
+  }
+
+  inner class BitmapTarget(private val urlDrawable: UrlDrawable) : SimpleTarget<Bitmap>() {
+    internal var drawable: Drawable? = null
+
+    override fun onResourceReady(resource: Bitmap, transition: Transition<in Bitmap>?) {
+
+      drawable = BitmapDrawable(context.resources, resource)
+      tvContents.post {
+        val w = tvContents.width
+        val hh = (drawable as BitmapDrawable).intrinsicHeight
+        val ww = (drawable as BitmapDrawable).intrinsicWidth
```

Comment:
This issue was mentioned earlier too. Rename these variables to something which become self-explanatory.

Reply:
rename these variables